samples/http_example/system-test/cloudiot_http_example.test.js

```diff
@@ -0,0 +1,148 @@
+// Copyright 2017 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+'use strict';
+
+const path = require('path');
+const {PubSub} = require('@google-cloud/pubsub');
+const assert = require('assert');
+const tools = require('@google-cloud/nodejs-repo-tools');
```

Comment:
we want to drop this

Reply:
there's been separate work happening to nuke references to `@google-cloud/nodejs-repo-tools`, there should be drop in replacements for most of the helpers it provides.

Use `execSync` rather than `tools.run`, for tests it's fine to use the `sync` helper :ok_hand: